packages/scoped-custom-element-registry/src/scoped-custom-element-registry.js

```diff
@@ -209,10 +212,10 @@ if (!ShadowRoot.prototype.createElement) {
 
   // Helper to create stand-in element for each tagName registered that delegates
   // out to the registry for the given element
-  const createStandInElement = (tagName, formAssociated) => {
+  const createStandInElement = (tagName) => {
     return class ScopedCustomElementBase {
-      static get formAssociated() {
-        return formAssociated;
+      static get ['formAssociated']() {
+        return true;
```

Comment:
So one other thing we should consider whether we can fix: Another side-effect of setting `true` here is that an element will show up in `form.elements`. I wonder if `Form.prototype.elements` is patchable to filter out CE's with `!formAssociated`....

Reply:
Haha, try as I may I cannot patch `Form.prototype.elements`, that's an [open issue on the polyfill right now](https://github.com/calebdwilliams/element-internals-polyfill/issues/43).

Reply:
TIL, an element is added to elements if `formAssociated` is true even if internals isn't set …

Reply:
I was thinking something like this: https://lit.dev/playground/?mods=gists#gist=be7e360bf80fb6431c1f5dc532063f4f

(the commented out code is roughly what I was thinking... the uncommented `return []` is just to prove that the `Form.prototype.elements` is indeed patchable...)

Reply:
@kevinpschaaf yeah, I've got a POC working, but one test is failing for some reason and there'll need to be several more tests written, but it's nearly there.